r/R/dplyr-collect.R

```diff
@@ -64,8 +64,8 @@ pull.Dataset <- function(.data,
   .data <- as_adq(.data)
   var <- vars_pull(names(.data), !!enquo(var))
   .data$selected_columns <- set_names(.data$selected_columns[var], var)
-  out <- dplyr::compute(.data)[[1]]
-  handle_pull_as_vector(out, as_vector)
+  out <- dplyr::compute(.data)
+  handle_pull_as_vector(out[[var]], as_vector)
```

Comment:
Doesn't matter, but this makes the fix more obvious

```suggestion
  out <- dplyr::compute(.data)[[var]]
  handle_pull_as_vector(out, as_vector)
```

Reply:
Definitely, not totally sure why I didn't do that at first 🙃